example/src/main/java/org/folio/tlib/example/Storage.java

```diff
@@ -0,0 +1,116 @@
+package org.folio.tlib.example;
```

Comment:
Cosmetics: renamed to `BookStorage` and move to `org.folio.tlib.example.storage`